### ISSUE TITLE
Clarify encryption setting

### ISSUE THREAD

**Issue by webframp** [MobileOrg/mobileorg#166]
Determine export requirements for encryption and if we can safely set:

```
<key>ITSAppUsesNonExemptEncryption</key><false/>
```

**Comment by mgmart** [comment]
If this is true, then not :(

http://stackoverflow.com/a/37619545

**Comment by webframp** [comment]
![](https://i.stack.imgur.com/Exatr.png)

![](https://i.stack.imgur.com/Bo1Rr.png)

Based on this I believe the answer is Yes, because the only way encryption is used falls under *(e)*: Limited to authentication, digital signature or the decryption of data or files.

**Comment by mgmart** [comment]
>  Limited to authentication, digital signature or the decryption of data or files.

But we also encrypt files, don't we?

**Comment by webframp** [comment]
> >  Limited to authentication, digital signature or the decryption of data or files.
> 
> But we also encrypt files, don't we?

If I understand the code right, then not as currently implemented, emacs org mode handles the encryption, MobileOrg needs the decryption key for display.

Our lives are easier if we stay compliant with *(a)* and *(e)* from that list.

**Comment by webframp** [comment]
See also: https://discussions.apple.com/thread/3228320?start=15&tstart=0

**Comment by mgmart** [comment]
Yeah, sure. If we really do not encrypt captures before transfer (haven't checked that) then we should overthink the whole encryption. ;)

**Comment by mgmart** [comment]
> See also: https://discussions.apple.com/thread/3228320?start=15&tstart=0

If I understood that correctly (only flown over) then symmetrical encryption is fine. Would make sense. Then it's a yes.

Remains the question what happens to captures if encryption is switched on :)

**Comment by webframp** [comment]
Details from [org-mobile.el](http://orgmode.org/cgit.cgi/org-mode.git/tree/lisp/org-mobile.el#n105)

In our codebase encryption appears to be used only on sync: https://github.com/MobileOrg/mobileorg/blob/71e01165eb5f9bb9d541d25009f45d8986bea9aa/Classes/Sync/SyncManager.m#L306

Not on capture: https://github.com/MobileOrg/mobileorg/blob/71e01165eb5f9bb9d541d25009f45d8986bea9aa/Classes/Capture/NewNoteController.m

**Comment by mgmart** [comment]
[v2 requirements](https://github.com/MobileOrg/mobileorg.next/blob/specification/docs/MobileOrgv2.org#encryption) must be taken in considerationd

**Comment by webframp** [comment]
libsodium would be my preferred crypto lib, two swift bindings: 
* https://github.com/drewcrawford/NaOH 
* https://github.com/jedisct1/swift-sodium

Doesn't address any gpg needs, but useful reference
